### ISSUE TITLE
About Artificial Intelligence and cryptocurrencies

### ISSUE THREAD

**Issue by tinysouth** [fireice-uk/xmr-stak#2766]
I think cryptocurrencies like Bitcoin and Litecoin are unnecessary, and the mining and earning of these currencies consume a large amount of human actual energy, resources, and computing power, resulting in strings that have no actual value or meaning. Using such strings as currency to measure value is not as good as using a certain indicator in the current development of artificial intelligence to act as cryptocurrency, measure value, and empower the development of artificial intelligence. It is recommended to replace traditional meaningless cryptocurrencies and redefine and develop cryptocurrencies using a certain indicator in the training of artificial intelligence generation models or a certain indicator during the operation of artificial intelligence

**Comment by tinysouth** [comment]
Keep up with the era of artificial intelligence to avoid being eliminated

**Comment by Shrey719** [comment]
?? This is a mining program, not a cryptocurrency. 
About wasting resources + energy, most people who dont want to use that much, like myself, use a solar powered mining rig.

**Comment by tinysouth** [comment]
Nearly.But I believe that utilizing computational power for large-scale AI models is much more worthwhile than applying it to cryptocurrencies, and could initiate a project that combines the characteristics of cryptocurrencies with AI.

**Comment by amlkyc** [comment]
> Nearly.But I believe that utilizing computational power for large-scale AI models is much more worthwhile than applying it to cryptocurrencies, and could initiate a project that combines the characteristics of cryptocurrencies with AI.

there is a good idea behind your words actually, I think someone should create a "mining" software to "mine" AI tokens, so computing power will be used to produce tokens for AI, still models will not be free, but it will be a lot cheaper than now and AI overall will become more accessable to the world. Some models are actually helpfull and can save people's time and drive progress. 

And also, about mining crypto, just dig into randomx algorith that monero uses, it's just trying to guess random values or something, it's not even calculating anything as I know, (that's why it's only suitable for cpu) so it's really pretty much waste of this power, but how whould you maintain crypto network other ways? It's sounds nonsense but I don't see it any other way, maybe something like "proof of stake" will work,  there is no wastings of power, but it's capitalism now xD

**Comment by PPPDUD** [comment]
@tinysouth I don't think that AI is somehow going to make mining _less_ wasteful. Proof-of-stake and proof-of-burn are more energy-efficient and don't cause much noise pollution.

Machine learning servers consume vast amounts of electricity, require lots of cooling, and are very noisy.

Furthermore, what do you mean by "certain indicators"?
